blocks/init/src/Blocks/custom/tab/tab-style.scss

```diff
@@ -0,0 +1,8 @@
+.block-tab {
+	display: none;
+	margin-bottom: 40px;
+
+	&--visible {
+		display: block;
+	}
```

Comment:
@goranalkovic-infinum you are an animation wizard, what do you recommend here for toggling the visible tab?